sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/fluent/models/ManagedClusterInner.java

```diff
@@ -652,33 +653,6 @@ public ManagedClusterInner withSupportPlan(KubernetesSupportPlan supportPlan) {
         return this;
     }
 
-    /**
-     * Get the enablePodSecurityPolicy property: (DEPRECATED) Whether to enable Kubernetes pod security policy
-     * (preview). PodSecurityPolicy was deprecated in Kubernetes v1.21, and removed from Kubernetes in v1.25. Learn more
-     * at https://aka.ms/k8s/psp and https://aka.ms/aks/psp.
-     * 
-     * @return the enablePodSecurityPolicy value.
-     */
-    public Boolean enablePodSecurityPolicy() {
-        return this.innerProperties() == null ? null : this.innerProperties().enablePodSecurityPolicy();
-    }
-
-    /**
-     * Set the enablePodSecurityPolicy property: (DEPRECATED) Whether to enable Kubernetes pod security policy
-     * (preview). PodSecurityPolicy was deprecated in Kubernetes v1.21, and removed from Kubernetes in v1.25. Learn more
-     * at https://aka.ms/k8s/psp and https://aka.ms/aks/psp.
-     * 
-     * @param enablePodSecurityPolicy the enablePodSecurityPolicy value to set.
-     * @return the ManagedClusterInner object itself.
-     */
-    public ManagedClusterInner withEnablePodSecurityPolicy(Boolean enablePodSecurityPolicy) {
-        if (this.innerProperties() == null) {
-            this.innerProperties = new ManagedClusterProperties();
-        }
-        this.innerProperties().withEnablePodSecurityPolicy(enablePodSecurityPolicy);
-        return this;
-    }
-
     /**
```

Comment:
This looks like a breaking change to me. We have this in 2.51.0.
https://github.com/Azure/azure-sdk-for-java/blob/azure-resourcemanager-containerservice_2.51.0/sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/fluent/models/ManagedClusterInner.java#L655-L680

Not sure why it's not detected by CI..

Reply:
This property was indeed removed in 2025-03-01. The detail info as following: https://github.com/Azure/azure-sdk-for-java/pull/45574/files#diff-aa3cfd63144668f7e10c04078627bfd9040cfd114e62d889b0b77abed17150dc